subsys/pm/pm.c

```diff
@@ -68,6 +68,20 @@ static inline void pm_state_notify(bool entering_state)
 	k_spin_unlock(&pm_notifier_lock, pm_notifier_key);
 }
 
+static inline int32_t ticks_expiring_sooner(int32_t kernel_ticks,
+					    int32_t low_latency_events_ticks)
```

Comment:
This function could be generic 
```suggestion
static inline int32_t ticks_expiring_sooner(int32_t ticks1, int32_t ticks2)
```

But we can generalize it when we find a new user

Reply:
Ok I am going to change it